### PR TITLE
docs: auth decorator with multiple strategies

### PR DESCRIPTION
**Pull request by jannyHou**
Update auth decorator docs to include multiple strategies. 
This adds docs for PR https://github.com/strongloop/loopback-next/pull/5735

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/Authentication-component-decorator.md [hunk 1]
```diff
@@ -73,7 +83,7 @@ export class WhoAmIController {
 An example of the decorator when options **are** specified looks like this:
 
 ```ts
-@authenticate('basic', { /* some options for the strategy */})
+@authenticate({'basic', { /* some options for the strategy */}})
```
Comment (raymondfeng) [inline docs/site/Authentication-component-decorator.md:86]: `{name: 'basic, options: { /* some options for the strategy */ }}`

Comment (jannyHou) [reply docs/site/Authentication-component-decorator.md:86]: good catch 👍

File: docs/site/Authentication-component-decorator.md [hunk 2]
```diff
@@ -18,15 +18,25 @@ functions with the
...
-```ts
-@authenticate(strategyName: string, options?: object)
-```
-
-or
-
-```ts
-@authenticate(metadata: AuthenticationMetadata)
-```
+- single strategy: `@authenticate(strategyName)`
```
Comment (dhmlau) [inline docs/site/Authentication-component-decorator.md:21]: Maybe we can group the single strategy with and without options together? likewise for the multiple strategies?

Comment (jannyHou) [reply docs/site/Authentication-component-decorator.md:21]: thanks @dhmlau changed.

**Pull request merged**